package.json

```diff
@@ -80,7 +80,7 @@
     "@types/tooltipster": "0.0.31",
     "@types/zxcvbn": "4.4.1",
     "codelyzer": "^5.2.2",
-    "cypress": "~4.12.1",
+    "cypress": "^9.3.1",
```

Comment:
I can't believe this works without breaking changes.
Please use ^-prefix only if you're sure cypress supports semantic versioning.